### ISSUE TITLE
"Error: Warning: message has unknown MIME version" causes fetching to stop

### ISSUE THREAD

**Issue by RienNeVaPlus** [freescout-helpdesk/freescout#2009]
The attached message stops fetching until the email is manually set to read from a client outside of freescout. The message seems to be spam; I've received two of them within the last week, both times crashing email fetching completely.
```
[2022-05-20 16:44:20] Warning: message has unknown MIME version
[2022-05-20 16:44:20] Folder: INBOX; Error: Warning: message has unknown MIME version
```

Some argue, the warning could be [misleading](https://stackoverflow.com/questions/32510660/bugs-and-issues-with-the-imapmailbox-php). And it might be a warning disguised as an error? ("Error: Warning:")

The email (anonymized):
```
Return-Path: <example@example.org>
Delivered-To: example@example.org
Received: from box.hostname ([127.0.0.1])
	by box.hostname with LMTP id AAAAHTxLwGAAAjWWW22mU1ig
	for <example@example.org>; Thu, 19 May 2022 10:06:12 +0200
X-Spam-Checker-Version: SpamAssassin 3.4.2 (2018-09-13) on box.hostname
X-Spam-Level: ****
X-Spam-Status: No, score=4.6 required=5.0 tests=BAYES_00,BOGUS_MIME_VERSION,
	DKIM_INVALID,DKIM_SIGNED,DMARC_NONE,FSL_BULK_SIG,HTML_MESSAGE,
	MIME_HTML_ONLY,RAZOR2_CF_RANGE_51_100,RAZOR2_CHECK,SPF_PASS,
	T_SCC_BODY_TEXT_LINE autolearn=no autolearn_force=no version=3.4.2
X-Spam-Report: 
	* -1.9 BAYES_00 BODY: Bayes spam probability is 0 to 1%
	*      [score: 0.0000]
	*  0.1 DMARC_NONE DMARC record not found
	* -0.1 SPF_PASS SPF check passed
	*  0.0 HTML_MESSAGE BODY: HTML included in message
	*  0.1 MIME_HTML_ONLY BODY: Message only has text/html MIME parts
	*  0.1 DKIM_SIGNED Message has a DKIM or DK signature, not necessarily
	*       valid
	*  1.9 RAZOR2_CF_RANGE_51_100 Razor2 gives confidence level above 50%
	*      [cf: 100]
	*  0.9 RAZOR2_CHECK Listed in Razor2 (http://razor.sf.net/)
	*  3.3 BOGUS_MIME_VERSION Mime version header is bogus
	*  0.1 FSL_BULK_SIG Bulk signature with no Unsubscribe
	* -0.0 T_SCC_BODY_TEXT_LINE No description available.
	*  0.1 DKIM_INVALID DKIM or DK signature exists, but is not valid
X-Spam-Score: 4.6
Received: from mail.hostname (mx1.hostname [217.86.191.57])
	(using TLSv1.2 with cipher ECDHE-RSA-AES128-GCM-SHA256 (128/128 bits))
	(No client certificate requested)
	by box.hostname (Postfix) with ESMTPS id 8CAFFFF0A0D
	for <example@example.org>; Thu, 19 May 2022 10:06:10 +0200 (CEST)
Authentication-Results: box.hostname; dmarc=none (p=none dis=none) header.from=preservarmeioambiente.com.br
Authentication-Results: box.hostname; spf=pass smtp.mailfrom=example@example.org
Authentication-Results: box.hostname;
	dkim=fail reason="signature verification failed" (2048-bit key; unprotected) header.d=preservarmeioambiente.com.br header.i=@preservarmeioambiente.com.br header.b="F42Jp0CY";
	dkim-atps=neutral
Resent-From: <example@example.org>
Received: from intern.hostname (192.168.1.31) by
 intern.hostname (192.168.1.31) with Microsoft SMTP Server
 (version=TLS1_2, cipher=TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256) id
 15.1.2375.18; Thu, 19 May 2022 10:06:09 +0200
Received: from inbound-52-29-100-147-eu-central-1.prod.hydra.sophos.com
 (52.29.100.147) by intern.hostname (192.168.1.31) with
 Microsoft SMTP Server (version=TLS1_2,
 cipher=TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256) id 15.1.2375.18 via Frontend
 Transport; Thu, 19 May 2022 10:06:09 +0200
Received: from ip-172-20-100-128.eu-central-1.compute.internal (ip-172-20-100-128.eu-central-1.compute.internal [127.0.0.1])
	by inbound-52-29-100-147-eu-central-1.prod.hydra.sophos.com (Postfix) with ESMTP id 4L3j9F4s2SzCqjK
	for <example@example.org>; Thu, 19 May 2022 08:06:09 +0000 (UTC)
Authentication-Results: mx-01-eu-central-1.prod.hydra.sophos.com; spf=softfail
 smtp.mailfrom=contato@preservarmeioambiente.com.br;
 dkim=pass header.d=preservarmeioambiente.com.br
 header.from=preservarmeioambiente.com.br
Received-SPF: softfail receiver=mx-01-eu-central-1.prod.hydra.sophos.com;
 client-ip=199.201.90.17; envelope-from=<contato@preservarmeioambiente.com.br>;
 helo=rev17.whmserver.net; 
X-Sophos-Product-Type: Gateway
X-Sophos-Email-ID: 38f95b782c0c4f29963cfbb758c34590
Received: from rev17.whmserver.net (rev17.whmserver.net [199.201.90.17])
 (using TLSv1.2 with cipher ECDHE-RSA-AES256-GCM-SHA384 (256/256 bits))
 (No client certificate requested)
 by mx-01-eu-central-1.prod.hydra.sophos.com (Postfix) with ESMTPS id
 4L3j8z0ljJzMkrm
 for <example@example.org>; Thu, 19 May 2022 08:05:54 +0000 (UTC)
DKIM-Signature: v=1; a=rsa-sha256; q=dns/txt; c=relaxed/relaxed;
 d=preservarmeioambiente.com.br; s=default; h=Date:Message-Id:From:
 Content-type:MIME-Version:Subject:To:Sender:Reply-To:Cc:
 Content-Transfer-Encoding:Content-ID:Content-Description:Resent-Date:
 Resent-From:Resent-Sender:Resent-To:Resent-Cc:Resent-Message-ID:In-Reply-To:
 References:List-Id:List-Help:List-Unsubscribe:List-Subscribe:List-Post:
 List-Owner:List-Archive; bh=edYcmu/sx5wWmn9ouXShCt/iyJ6pW6rqPYfbD+NpDyQ=; b=F
 42Jp0CYX6b858TOWi45gOHT0rMx++2kdtw6t/V0zsr1vay2dIPOTkx0bos42Z/jJemEdjCvE0VZ2x
 xIhMqBDZtRQrx3eMifmpJ2RlAx71VquV4UgMhY+zrmP0iUNilXnbZD9ns+BAA7m2m8d5dreXJbuj+
 F93BKOSJLbH38ay++pFyDWwJt1y6jbHAIxxjXrH/tzvNEmPmzXTRL/6juevEpbh5JAXbNKoKWiNkd
 SsmnwstQ1hXM8qFcxVzp+HHJGxP+wUVLwJ0R+oOkefyXi12PHycX8jQdzhpsDfiVTGX9G96HY5qmm
 bzTCuawBfpDeAmS8f1QVvO5Q3rsgdILAA==;
Received: from pma by host12.whmserver.net with local (Exim 4.95)
 (envelope-from <contato@preservarmeioambiente.com.br>)
 id 1nrbAO-00GEXX-SR for example@example.org;
 Thu, 19 May 2022 05:05:52 -0300
To: example@example.org
Subject: Fale Conosco Preservar Arqueologia e Meio Ambiente
X-PHP-Script: www.preservarmeioambiente.com.br/index.php for 193.187.106.213
X-PHP-Originating-Script: 1753:envia-contato.php
MIME-Version: 1.1
Content-type: text/html; charset=utf-8
From: Preservar Arqueologia e Meio Ambiente
 <contato@preservarmeioambiente.com.br>
Message-Id: <E1nrbAO-00GEXX-SR@host12.whmserver.net>
Date: Thu, 19 May 2022 05:05:52 -0300
X-AntiAbuse: This header was added to track abuse,
 please include it with any abuse report
X-AntiAbuse: Primary Hostname - host12.whmserver.net
X-AntiAbuse: Original Domain - example.org
X-AntiAbuse: Originator/Caller UID/GID - [1753 501] / [47 12]
X-AntiAbuse: Sender Address Domain - preservarmeioambiente.com.br
X-Get-Message-Sender-Via: host12.whmserver.net: authenticated_id: pma/from_h
X-Authenticated-Sender: host12.whmserver.net: contato@preservarmeioambiente.com.br
X-Source:
X-Source-Args:
X-Source-Dir: preservarmeioambiente.com.br:/public_html
X-Sophos-Email: [eu-central-1] Antispam-Engine: 5.0.0,
 AntispamData: 2022.5.19.73919
X-Sophos-SenderHistory: ip=199.201.90.17, fs=498690, fso=498690, da=146406174,
 mc=12, sc=0, hc=12, sp=0, re=0, sd=0, hd=1
X-LASED-SpamProbability: 0.151100
X-LASED-Hits: BLOGSPOT_URI 0.050000, BODYTEXTH_SIZE_10000_LESS 0.000000,
 BODY_SIZE_2000_2999 0.000000, BODY_SIZE_5000_LESS 0.000000,
 BODY_SIZE_7000_LESS 0.000000, BR_IN_HDRS 0.000000, CANPHARM_SUBJ_PILL 0.050000,
 CTYPE_JUST_HTML 0.848000, DKIM_ALIGNS 0.000000, DKIM_SIGNATURE 0.000000,
 DQ_S_H 0.000000, FROM_NAME_PHRASE 0.000000, HAS_X_PHP_SCRIPT 0.000000,
 HTML_NO_HTTP 0.100000, INBOUND_SOPHOS 0.000000, NO_FUR_HEADER 0.000000,
 OBFU_SHORT_10CHARS 0.500000, SINGLE_URI_IN_BODY 0.000000,
 WEBMAIL_SOURCE 0.000000, WEBMAIL_X_PHP_SCRIPT 0.000000, __ANY_URI 0.000000,
 __AUTH_RES_DKIM_PASS 0.000000, __BITCOIN_BODY 0.000000,
 __BODY_NO_MAILTO 0.000000, __BODY_TEXT_X4 0.000000, __CP_URI_IN_BODY 0.000000,
 __CT 0.000000, __CTYPE_HTML 0.000000, __CTYPE_IS_HTML 0.000000,
 __DKIM_ALIGNS_1 0.000000, __DKIM_ALIGNS_2 0.000000, __DQ_IP_FSO_LARGE 0.000000,
 __DQ_S_HIST_1 0.000000, __DQ_S_IP_1MO 0.000000, __DQ_S_IP_MC_10_P 0.000000,
 __DQ_S_IP_MC_1_P 0.000000, __DQ_S_IP_MC_5_P 0.000000, __DQ_S_IP_RE_0 0.000000,
 __DQ_S_IP_RE_49_L 0.000000, __DQ_S_IP_RE_4_L 0.000000,
 __DQ_S_IP_RE_99_L 0.000000, __DQ_S_IP_RE_9_L 0.000000,
 __DQ_S_IP_SP_0_P 0.000000, __FRAUD_ANTIABUSE 0.000000,
 __FRAUD_MONEY_CURRENCY 0.000000, __FRAUD_MONEY_CURRENCY_POUND 0.000000,
 __FRAUD_SOURCE_DIR 0.000000, __FROM_NAME_NOT_IN_BODY 0.000000,
 __HAS_FROM 0.000000, __HAS_HTML 0.000000, __HAS_MSGID 0.000000,
 __HAS_X_PHP_ORIG_SCRIPT 0.000000, __HAS_X_PHP_SCRIPT 0.000000,
 __HEADER_ORDER_FROM 0.000000, __HIGHBITS 0.000000, __HTML_TAG_TABLE 0.000000,
 __HTTPS_URI 0.000000, __MIME_HTML 0.000000, __MIME_HTML_ONLY 0.000000,
 __MIME_TEXT_H 0.000000, __MIME_TEXT_H1 0.000000, __MIME_VERSION 0.000000,
 __SANE_MSGID 0.000000, __SINGLE_URI_HTML 0.000000,
 __STYLE_RATWARE_NEG 0.000000, __STYLE_TAG 0.000000, __SUBJ_ALPHA_END 0.000000,
 __TAG_EXISTS_HTML 0.000000, __TO_MALFORMED_2 0.000000, __TO_NO_NAME 0.000000,
 __URI_IN_BODY 0.000000, __URI_MAILTO 0.000000, __URI_NOT_IMG 0.000000,
 __URI_NO_PATH 0.000000, __URI_NO_WWW 0.000000, __URI_NS 0.000000,
 __URI_WITHOUT_PATH 0.000000, __X_PHP_ORIG_UID 0.000000
X-LASED-Impersonation: False
X-LASED-Spam: NonSpam
X-Sophos-MH-Mail-Info-Key: NEwzajlGNHMyXawd3XXXe333333TE3Mi4yMC4xMDAuMTI4

<html><head><meta http-equiv=Content-Type content=text/html; charset=utf-8 /><title>Preservar Arqueologia e Meio Ambiente</title></head><body><table width='570'>    <tr>    	<td>        	<table width='100%'>                <tr>                	<td class='CinzaM' style='font-size:14px;'><b>Fale Conosco</b></td>                </tr>            </table>    	</td>    </tr>    <tr>    	<td>        	<table width='100%'>                <tr>                	<td class='CinzaM' style='font-size:14px;'><b>Coloperasd,</b></td>                </tr>            </table>    	</td>    </tr>    <tr>    	<td>        	<table width='100%'>                <tr>                	<td class='CinzaM' style='font-s
 ize:14px;'>
							Sua solicitação de contato foi recebida pela equipe Preservar Arqueologia e Meio Ambiente.<br>
							Em breve entraremos em contato.
							<br><br>
							Veja abaixo os detalhes da mensagem.
							<br><br>
							<b>Data da mensagem:</b> 19/05/2022 05:05:52.<br>
							<b>Nome:</b> Coloperasd.<br>
							<b>Email:</b> example@example.org.<br>
							<b>Telefone:</b> 89456985236.<br>
							<b>Como nos achou?:</b> .<br>
							<b>Assunto:</b> Investieren Sie in Bitcoin.<br>
							<b>Mensagem:</b>  
 
Die Krise? Wir haben nicht!! 
Warte nicht ! Verdienen Sie mit uns! https://spam-url.host .<br>  
							</td>                </tr>            </table>    	</td>    </tr>    <tr>    	<td>        	<table width='100%'>                <tr>                	<td class='CinzaM' style='font-size:14px;'>
							Atenciosamente,<br>
							<b>
							Equipe Preservar Arqueologia e Meio Ambiente<br>
							contato@preservarmeioambiente.com.br<br>
							</b>
							<br><br>
							Email de disparo automático,<br>
							<b>favor não responder.</b>
							</td>                </tr>            </table>    	</td>    </tr>    <tr>    	<td><br></td>    </tr></table></body></html>

```

Thanks

**Comment by freescout-helpdesk** [comment]
Are you using a modern Microsoft Exchange authentication or regular IMAP?

**Comment by RienNeVaPlus** [comment]
We are using regular IMAP

**Comment by bfren** [comment]
I had this error message too, but it was fixed by switching to PHP 7.4 instead of PHP 8.

**Comment by RienNeVaPlus** [comment]
Thanks @bfren, we are indeed on PHP 8.

**Comment by freescout-helpdesk** [comment]
The issue with fetching stopping was fixed in FreeScout v1.8.15

**Comment by freescout-helpdesk** [comment]
Another fix was added in 1.8.16 release.
